main.go

```diff
@@ -42,6 +43,11 @@ func main() {
 	flag.Parse()
 	flags.args = flag.Args()
 
+	if *printVersion {
+		fmt.Printf("moq version %s\n", version)
```

Comment:
Perhaps we should just do `fmt.Println(version)` - allows for programmatic use too?

Reply:
As far as I can tell, it is pretty standard for a tool to print not only its version but also its name. I quickly tried with `docker`, `kubectl`, `terraform` and some other tools and all of them provide the name as well. The same holds true also for the `go` command it self. Therefore I am in favor of keeping the name in there.
That being said, if you insist on having only the version printed, I will update the PR of course.

Reply:
OK then - no need to be different for the sake of it.